java/debugger.jpda.truffle/external/binaries-list

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+FC9F6A53E0941C957779B2488E0AC3339198C623 graal-sdk-1.0.0-rc7-20180922.014034-77.jar
```

Comment:
Is it possible to use Maven co-ordinates instead? Release Candidate 6 bits are on [maven central](https://search.maven.org/search?q=g:com.oracle.truffle).

Reply:
I'll need to verify if RC6 is sufficient...